src/native/_nfs.js

```diff
@@ -53,22 +53,6 @@ const readFileInfo = (fileInfo) => {
         ? new Buffer(0)
         : new Buffer(ref.reinterpret(file.user_metadata_ptr, file.user_metadata_len));
 
-  if (user_metadata_ptr) {
```

Comment:
Why remove this branch?
The addition of error handling at https://github.com/maidsafe/safe_app_nodejs/pull/252/files#diff-c2284895deccc698165325b396ad79b9R93, creates a decrease in code coverage however I'm not able to create a testing condition where `app_reconnect` throws an error in mock routing.
Thus my search for a way to increase coverage to make coveralls happy.
The above block of code is safe to remove because the condition can never be met since `user_metadata_ptr` will always be an instance of `Buffer`.
cc: @bochaco